src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.commands;
+
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.ShooterTable;
+
+import java.util.TreeMap;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ShooterCommand extends CommandBase{
+    Shooter m_shooter;
+    Intake m_intake;
+    TreeMap <Double, ShooterTable> m_speedTable = new TreeMap <Double, ShooterTable> (){}; //set up lookup table for shooter speeds
+
+    public ShooterCommand(Shooter shooter, Intake intake){
```

Comment:
The ShooterCommand will need the Vision subsystem passed in so it ca determine the distance to the target.